msi/GitProduct.wxs

```diff
@@ -278,15 +282,20 @@
                   Execute="deferred" Return="ignore" Impersonate="no" />
 
     <InstallExecuteSequence>
-      <!-- Execute the post-install.bat at the end of the installation
-           transaction on install and repair but not removal. -->
-      <Custom Action="ExecPostInstallBat" Before="InstallFinalize">
+    <?if $(sys.BUILDARCH)=x64 ?>
+      <Custom Action="ExecConfigurePackSizeLimit" Before="ExecConfigureSslCAInfo">
+        <![CDATA[REMOVE <> "ALL"]]>
+      </Custom>
+    <?endif?>
+      <Custom Action="ExecConfigureSslCAInfo" Before="ExecConfigureCoreAutoCrlf">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
-      <Custom Action="ExecConfigureCoreAutoCrlf" After="ExecPostInstallBat">
+      <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecPostInstallBat">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
-      <Custom Action="ExecConfigureSslCAInfo" After="ExecConfigureCoreAutoCrlf">
+      <!-- Execute the post-install.bat at the end of the installation
+           transaction on install and repair but not removal. -->
+      <Custom Action="ExecPostInstallBat" Before="InstallFinalize">
```

Comment:
Why is this represented as added lines? I guess there was a reordering going on... It would be nice to rewrite the commits (via `rebase -i`) such that the correct spelling and the correct order is used from the get-go, so that this change really only adds lines, instead of also moving others.